src/editing.md

```diff
@@ -1,463 +1,489 @@
-underline work like they do in a word processing program. The next two
-buttons allow you to subscript or superscript text, which is useful for
-chemical compounds like H<sub>2</sub>O or simple mathematical equations like
-x<sup>2</sup>.
+Die meisten Bedienelemente können auch über Tataturkürzel aktiviert werden.
+Sie werden angezeigt, wenn du mit der Maus darüber fährst.
 
-The rubber eraser button clears any formatting in the currently selected text — including the colour
-of the text,  whether the selected text is bold, etc.
+Anki behält beim Einfügen von Inhalten über die Zwischenablage im Normalfall
+den Großteil der Formatierungen bei. Wenn du dabei jedoch die Umschalttaste
+gedrückt hältst, werden die Inhalte ohne Formatierung eingefügt.
+In den Einstellungen kannst du dieses Verhalten mit der Option
+"Text aus der Zwischenablage ohne Formatierung einfügen" zum Standard machen.
 
-The next three buttons allow creating lists, text alignment and text indent.
-Then, there are two buttons to allow you to change text colour.
+## Lücken einfügen
 
-The \[…​\] button is visible when a cloze note type is selected.
+In Lückentexten werden ein oder mehrere Wörter eines Satzes ausgeblendet.
+Nehmen wir den folgenden Satz:
 
-You can use the paper-clip button to select audio, images, and videos from
-your computer's hard drive and attach them to your notes. Alternatively, you
-can copy the media onto your computer's clipboard (for instance, by
-right-clicking an image on the web and choosing 'Copy Image') and paste
-it into the field that you want to place it in. For more information
-about media, please see the [media](media.md) section.
+    Die Universität Wien wurde 1365 gegründet.
 
-The microphone icon allows you to record from your computer's microphone
-and attach the recording to the note.
+Machen wir "1365" zu einer Lücke, würde der Satz beim Wiederholen so aussehen:
 
-The Fx button shows shortcuts to add MathJax or
-[LaTeX](math.md) to your notes.
+    Die Universität Wien wurde […] gegründet.
 
-The `</>` button allows editing the underlying HTML of a field.
+"1365" ist auf der Vorderseite ausgeblendet und wird auf der Rückseite
+aufgedeckt.
 
-Anki 2.1.45+ supports adjusting sticky fields directly from the editing screen.
-If you click on the pin icon on the right of a field, Anki will not clear out
-the field's content after a note is added. If you find yourself entering the
-same content into multiple notes, you may find this useful. On previous Anki
-versions, sticky fields were toggled from the Fields screen.
+Siehe Regel 5 [hier](http://supermemo.org/deutsch/articles/20rules.htm),
+um mehr über die Vorteile von Lückentexten zu erfahren.
 
-Most of the buttons have shortcut keys. You can hover the mouse cursor
-over a button to see its shortcut.
+Anki bietet einen eigenen Notiztyp, der die Erstellung von Lückentexten
+leicht macht. Um einen Lückentext zu erstellen, wähle zuerst den Notiztyp
```

Comment:
```suggestion
Anki bietet einen eigenen Notiztyp, um einfach Lückentexte zu erstllen.
Um einen Lückentext zu erstellen, wähle zuerst den Notiztyp
```

Reply:
*erstellen
I'll fix that one later too.